internal/connect/hwinfo.go

```diff
@@ -26,6 +26,7 @@ type hwinfo struct {
 	Hostname      string `json:"hostname"`
 	Cpus          int    `json:"cpus"`
 	Sockets       int    `json:"sockets"`
+	Clusters      int    `json:"clusters"`
```

Comment:
should we be sending the json to scc? Should this be `json:"-"` ?

Reply:
not sure. it doesn't hurt. just another bit of info on the systems. SCC can use that if they feel they need it.